model_compression_toolkit/quantizers_infrastructure/keras/README.md

```diff
@@ -8,11 +8,11 @@ Trainable quantizer can be Weights Quantizer or Activation Quantizer.
-   - Implement [`initialize_quantization`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py#L71) where you can define your parameters for the quantizer.
-   - Implement [`__call__`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py#L88) method to quantize the given inputs while training. This is your custom quantization itself. 
-   - Implement [`convert2inferable`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py#136) method. This method exports your quantizer for inference (deployment). For doing that you need to choose one of our Inferable Quantizers ([Weights Inferable Quantizers](https://github.com/sony/model_optimization/tree/main/model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/weights_inferable_quantizers), [Activation Inferable Quantizers](https://github.com/sony/model_optimization/tree/main/model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/activation_inferable_quantizers) according to target when implement `convert2inferable`, and set your learned quantization parameters there. 
+     - [`QuantizationTarget`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_inferable_quantizer.py): Enum indicating weights or activations.
+     - [`QuantizationMethod`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/core/common/target_platform/op_quantization_config.py): List of quantization methods (Uniform, Symmetric, etc.)
+   - Implement [`initialize_quantization`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) where you can define your parameters for the quantizer.
+   - Implement [`__call__`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) method to quantize the given inputs while training. This is your custom quantization itself. 
+   - Implement [`convert2inferable`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) method. This method exports your quantizer for inference (deployment). For doing that you need to choose one of our Inferable Quantizers ([Inferable Quantizers](https://github.com/sony/model_optimization/tree/main/model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/README.md)) according to target when implement `convert2inferable`, and set your learned quantization parameters there. 
```

Comment:
I think the link should be to the directory of the quantizers, no?

Reply:
No. to the readme.